assets/src/components/app.tsx

```diff
@@ -100,6 +91,19 @@ export const AppRoutes = () => {
                   element={<ShuttleMapPage />}
                 />
                 <BrowserRoute path="/settings" element={<SettingsPage />} />
+              </Route>
+              <Route
+                element={
+                  <RouteElement
+                    selectedVehicleOrGhost={selectedVehicleOrGhost}
+                    openView={openView}
+                    openMapEnabled={false}
+                    vppTabMode={vppTabMode}
+                    setTabMode={setTabMode}
+                    closeView={closeView}
+                  />
+                }
+              >
```

Comment:
question: what's the rationale of splitting this(`RouteElement`) out into it's own component and putting `path={mapMode.path}` into it's own `Route`?

Reply:
They're split out into separate `<Route/>`s to allow different values of `openMapEnabled` to be set. Splitting out `<RouteElement/>` just seemed natural because otherwise I'd be duplicating the logic in there about the placement of the `<Outlet/>`, conditional rendering of `<RightPanel/>`, etc.

Reply:
oooooh that's super clean, I like it!